project.clj

```diff
@@ -4,6 +4,8 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.7.0"]
+                 [org.clojure/tools.reader "0.10.0"]
+                 [org.clojure/data.json "0.2.6"]
```

Comment:
Gentle preference for [Cheshire](https://github.com/dakrone/cheshire) over data.json. Not a blocker -- either a "tweak" or a "maybe lets make a note to switch later".